kubeflow/jupyter/NOTEBOOK.MD

```diff
@@ -0,0 +1,121 @@
+## Creating a Jupyter notebook in user-defined namespaces
+In order to create a notebook in a given namespace, Kubeflow expects a service account named `jupyter-notebook`, whcih allows K8s resource access,  and a secret named `user-gcp-sa,` which allows GCP service access, to exist in the namespace. 
```

Comment:
If ```jupyter-notebook``` is used by controller only, we might not need it in user namespace?
Instead we can give ```jupyter-notebook``` cluster-level permission to create notebooks?

Reply:
`jupyter-notebook` service account is used by notebooks (not the controller) to access K8s resources (e.g., create pod). Notebooks use this service account to run Fairing.

Reply:
Got it, can we use namespace default service account?

Reply:
You are right. I guess for now we can create default service accounts for all namespaces. However, [in this issue](https://github.com/kubeflow/kubeflow/issues/2992), we would like to automate injection of service accounts  so that users have options to inject their desired service accounts.